src/components/Calculator.jsx

```diff
@@ -1,34 +1,49 @@
-                    <button className='btn' type='button'>8</button>
-                    <button className='btn' type='button'>9</button>
-                    <button className='btn btn-calc' type='button'>X</button>
-                    <button className='btn' type='button'>4</button>
-                    <button className='btn' type='button'>5</button>
-                    <button className='btn' type='button'>6</button>
-                    <button className='btn btn-calc' type='button'>-</button>
-                    <button className='btn' type='button'>1</button>
-                    <button className='btn' type='button'>2</button>
-                    <button className='btn' type='button'>3</button>
-                    <button className='btn btn-calc' type='button'>+</button>
-                    <button className='btn btn-zero' type='button'>0</button>
-                    <button className='btn' type='button'>.</button>
-                    <button className='btn btn-calc' type='button'>=</button>
+                    <button className='btn btn-oper' type="button" onClick={this.handleClick}> AC </button>
+                    <button className='btn btn-oper' type="button" onClick={this.handleClick}> +/- </button>
+                    <button className='btn btn-oper' type="button" onClick={this.handleClick}>%</button>
+                    <button className='btn btn-calc' type="button" onClick={this.handleClick}>÷</button>
+                    <button className='btn' type="button" onClick={this.handleClick}>7</button>
+                    <button className='btn' type='button' onClick={this.handleClick}>8</button>
+                    <button className='btn' type='button' onClick={this.handleClick}>9</button>
+                    <button className='btn btn-calc' type='button' onClick={this.handleClick}>x</button>
+                    <button className='btn' type='button' onClick={this.handleClick}>4</button>
+                    <button className='btn' type='button' onClick={this.handleClick} >5</button>
+                    <button className='btn' type='button' onClick={this.handleClick} >6</button>
+                    <button className='btn btn-calc' type='button' onClick={this.handleClick} >-</button>
+                    <button className='btn' type='button' onClick={this.handleClick} >1</button>
+                    <button className='btn' type='button' onClick={this.handleClick} >2</button>
+                    <button className='btn' type='button' onClick={this.handleClick} >3</button>
+                    <button className='btn btn-calc' type='button' onClick={this.handleClick} >+</button>
+                    <button className='btn btn-zero' type='button' onClick={this.handleClick} >0</button>
+                    <button className='btn' type='button' onClick={this.handleClick} >.</button>
+                    <button className='btn btn-calc' type='button' onClick={this.handleClick} >=</button>
```

Comment:
- [ ] [Optional] Kindly make the design look like what we have in the design guide, I believe you are trying to do something different which is very good, but it is more professional to stick to a given design :+1: